packages/office-ui-fabric-react/src/components/Layer/examples/Layer.Customized.Example.tsx

```diff
@@ -67,19 +61,19 @@ export class LayerCustomizedExample extends React.Component<{}, ILayerCustomized
     );
   }
 
-  private _onDismissPanel = (): void => {
+  private _closePanel = () => {
```

Comment:
Why this name change? At least to me it doesn't seem like a bad thing to have the callback functions be named similarly to the props they're used for.

Reply:
Yeah names are hard, I'm aiming for consistency more than anything here.  There were some named _onDismissPanel, some with hide, some with close.  I only used `_closePanel` for consistency, but I'm not opinionated about it at all.  If anything I'd agree with you that the name should mirror the action happening, so then if `onDismissed` is the prop then options are `_onDismissed`, `_onDismissedPanel`, `_onDismissPanel`, or simply `_hidePanel`.  I'd say `_hidePanel`  probably reflects the action being taken most semantically.  Thoughts?